src/kernels/gpu_reference_kernel/naive_conv.cpp

```diff
@@ -119,6 +119,8 @@ inline __device__ __host__ int8_t cast_to(const int32_t& val)
 template <bool ASSUME_PACKED, typename src_data_t, typename acc_data_t, typename dst_data_t>
 inline __device__ void naive_conv_fwd_nchw(const src_data_t* __restrict__ p_in,
                                            const src_data_t* __restrict__ p_wei,
+                                           const float alpha,
```

Comment:
The type of `alpha/beta` for naive kernel should be `accumulator type` as that's `double` for floating point inputs and `int32_t` for integer inputs sucha as `int8_t`.